features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store/source/src/app/data/Subscription.js

```diff
@@ -0,0 +1,70 @@
+/**
+ * Copyright (c) 2018, WSO2 Inc. (http://wso2.com) All Rights Reserved.
```

Comment:
According to this https://github.com/wso2/carbon-apimgt/pull/5334#discussion_r182550628 discussion, we no longer add copyright year in the license header .
@nuwand , Please correct me if it's no the case.